fiscal_epos_print/models/account_journal.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    fiscalprinter_payment_type = fields.Selection(
+        [
+            ('0', 'Cash'),
+            ('1', 'Cheque'),
+            ('2', 'Not collected / Electronic payment'),
+            ('3', 'Ticket')
+        ],
+        'Payment type',
+        help='The payment type to send to the Fiscal Printer.',
+        default='0'
+    )
+
+    fiscalprinter_payment_index = fields.Integer(
+        string='Credit Card/Ticket Index',
+        help='Set the index of the given payment type to specify the detail. '
+             'Such index of the payment type must programmed on the fiscal '
+             'printer')
```

Comment:
@eLBati @GSLabIt 
Visto quanto detto in https://github.com/OCA/l10n-italy/pull/1557#discussion_r366776298 non mi è chiaro se anche qui "Credit Card/Ticket Index" vada sostituito con "Electronic Payment/Ticket Index".

Reply:
Direi di sì, modificato